lib/src/main/java/de/edux/augmentation/effects/OtsuThresholdingAugmentation.java

```diff
@@ -0,0 +1,85 @@
+package de.edux.augmentation.effects;
+
+import de.edux.augmentation.core.AbstractAugmentation;
+
+import java.awt.*;
+import java.awt.image.BufferedImage;
+
+public class OtsuThresholdingAugmentation extends AbstractAugmentation {
```

Comment:
This class is public API, pls provide a good JavaDoc Documentation at class and public methods.

Reply:
javaDocs added also refactor methods from static.